cypress/e2e/BasePage.pageObject.ts

```diff
@@ -1,76 +1,42 @@
 export class BasePage {
+  protected outerElementHTMLElement: HTMLElement;
+  protected innerElementHTMLElement: HTMLElement;
+
+  constructor({ outerElement, innerElement }: { outerElement: HTMLElement; innerElement: HTMLElement }) {
+    this.outerElementHTMLElement = outerElement;
+    this.innerElementHTMLElement = innerElement;
+    this.bodyElement.then(el => el[0].style.height = "600px");
```

Comment:
```suggestion
    this.bodyElement.then(el => el.css('height', '600px'));
```

Btw. Wouldn't it be better to change the viewport via cypress API?

Reply:
Viewport is iframe so it is visually okay but still bigger than `body` which is same height as my scrollbar list